sigmastate/src/main/scala/sigmastate/lang/syntax/Basic.scala

```diff
@@ -1,42 +1,40 @@
 package sigmastate.lang.syntax
 
-import fastparse.all._
+import fastparse._; import NoWhitespace._
 import fastparse.CharPredicates._
 import scalan.Nullable
 import sigmastate.lang.SourceContext
 import sigmastate.lang.exceptions.SigmaException
 
 object Basic {
```

Comment:
Seems like proper time to add ScalaDoc to Basic

Reply:
this files have been moved on restructuring PR. Added docs on top of the latest PR here https://github.com/ScorexFoundation/sigmastate-interpreter/commit/9270d42c36b83c53b54748caffc9ec09a742acb5